controllers/component_build_controller_pac_repository.go

```diff
@@ -225,6 +225,14 @@ func generatePACRepository(component appstudiov1alpha1.Component, config *corev1
 		gitProviderConfig.URL = url
 	}
 
+	if gitProviderConfig != nil {
+		repositoryGitProvider := gitProvider
+		if repositoryGitProvider == "bitbucket" {
+			repositoryGitProvider = "bitbucket-cloud"
```

Comment:
Could you add a comment with a link to some documentation that would explain why `bitbucket-cloud` is the correct value?

Reply:
added

Reply:
nitpick: https://pipelinesascode.com/docs/guide/incoming_webhook/#incoming-webhook-url looks like the more relevant section

Reply:
updated

Reply:
@rcerven naive query 
may be i m missing something 
PAC already support bitbucket-cloud as per the doc https://pipelinesascode.com/docs/guide/incoming_webhook/#incoming-webhook-url any reason for overriding it :thinking:

Reply:
@savitaashture PAC requires 'bitbucket-cloud' string, but in build-service we are using 'bitbucket' that is why I am converting it to string which PAC understands

Reply:
Okay got it thank you